designer/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM node:12-alpine3.12 AS build
 WORKDIR /usr/src/app
 COPY . .
 RUN apk add --no-cache bash
+RUN npm install -g typescript
+RUN npm install -g @babel/core @babel/cli
```

Comment:
we can think about optimising our builds + images eventually.. (ie having everything installed + cached for the next container)